### PR TITLE
Responsive Web Design

### PR DESCRIPTION
**Pull request by chukwuemeka1234**
This project is the clone of The Next Web

- HTML and CSS were used to build this project.
- Media query was used in CSS for the Mobile, Tablet, and Desktop versions for a Responsive Web Design

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
**Hello guys, I'm so happy to see you reach this far, your project is really good**💪✔
Let's just fix few issues and move.
### required changes.

1. Fix the tablet breakpoint (Hint: you can use flex: 1 on the sub-content-container class)
**Highlights section**

1. Fix image problems(no image is displayed) see image.
Highlights-missing-image

2. The title should be a link (TNW).

**Latest funding**

1. The title should be a link
2. The nav menu (See more, Get updates, etc.) should be grouped in a nav tag, and each one should have a link (a tag)

**The header logo**
The logo should be a link.

**footer**
Increase the font for the logo in the footer, see from image;
footer-logo
**Remember** "As described in the Code reviews limit Policy you have 2 code reviews left for this project."

Make these changes and submit another review request.💪💪

Just in case you have a question, please reach me on slack by searching elisha kyakopo
Cheers🎉👈

**Comment by elisha2kyakpo1** [comment]
Hello guys, remember also to update your readme file, some info is missing, like your details. Happy coding

**Comment by chukwuemeka1234** [comment]
Thank you for the review.

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
 # the-next-web-clone
+
+Responsive Web Design
+
+<h2><b>Week 2 Media query project</b></h2>
+<hr>
+<p>The assignment is the Responsive Web Design</p>
+<hr>
+<img src="./img/the-next-web-screen-shoot.JPG">
```
Comment (Berabjesus) [inline README.md:9]: - [ ] As requested by the previous reviewer, some details, i.e the image on the readme, is not showing. Please fix that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+                </div>
+                <div class="links-right">
+                    <a href="#">About</a>
+                    <a href="#">Advertise</a>
+                    <a href="#">Jobs</a>
+                    <a href="#">Contact</a>
+                </div>
+            </div>
+            <div class="header-nav-2">
+                <div class="logo-wrap"><img class="logo-image" src="img/tnw-logo.png" alt="logo-image"></div>
```
Comment (Berabjesus) [inline index.html]: - [ ] make sure the logo has a link (`a` tag) on all version where its visible

File: index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+                    </p>
+                </a>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="content-container flex">
+            <div class="content-info first-image flex flex-1">
+                <a href="#"><img src="img/logo-transparent.png" alt="big logo"></a>
+                <h1>Opal cuts off your apps’ internet access to help you stop doomscrolling</h1>
```
Comment (Berabjesus) [inline index.html]: - [ ] Titles should be links. The title of each article is a header (`hx` tag) and has a link (`a` tag)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+                <span class="date">Tuesday - January 26, 2021</span>
```
Comment (Berabjesus) [inline index.html:65]: - [ ] [Optional] This text is not fully visible. perhapse you could add some styling to it. Example `padding: 5px 0 0 
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+                <span class="invisible-links">
+                    <a href="#">LATEST</a>
+                    <a href="#">HARD FORK</a>
+                    <a href="#">PLUGGED</a>
+                    <a href="#">README</a>
+                    <a href="#">GROWTH QUARTERS</a>
+                </span>
+                <div class="">
+                    <a href="#"><i class="fas fa-search"></i></a>
+                    <a href="#"><i class="fas fa-bars"></i></a>
```
Comment (Berabjesus) [inline index.html]: - [ ] [optional] The header on your mobile version needs to have a hamburger icon. Currently, on mobile width of `360 - 400 px` it doesn't show up.
### your page at 384 px 👇 
image

### original website 👇 
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+            </div>
+            <div class="footer-links">
+                <ul>
+                    <li><a href="#">EVENTS</a></li>
+                    <li><a href="#">ABOUT</a></li>
+                    <li><a href="#">ADVERTISE</a></li>
+                    <li><a href="#">JOBS</a></li>
+                    <li><a href="#">CONTACT</a></li>
+                </ul>
+            </div>
```
Comment (Berabjesus) [inline index.html]: - [ ] [Optional] The highlighted items should have a proper text size
### your page👇 
image

### original page👇 

image

**Review by Berabjesus — CHANGES_REQUESTED** [review]
## Status: CHANGES REQUIRED :warning: :construction: 
#### Hi @mahbubul14  a very awesome start :+1:  Good job with what you have done so far. 👍 

#### However, you do have a few changes to carry out before we proceed so please take a look at the things you have to work on and their respective  _review comments_ below.

#### Mandatory 

- [ ] As requested by the previous reviewer, The logo should be a link. That is the TNW logo should be a link on the mobile version too.
- [ ] Titles should be links. The title of each article is a header (`hx` tag) and has a link (`a` tag)
- [ ] Some details (screenshots) are not showing on your Readme. 

#### Optional 

- [ ] remove everything in your .github folder except the workflows folder

- [ ] Some text (the date) in the `content-info` class is not showing properly. 

- [ ] The header on your mobile version needs to have a hamburger icon. Currently, on mobile width of `360 - 400 px` it doesn't show up.
- [ ] Footer link texts don't have proper formatting.

**Please have a look at the specific comments below**

#### ❗ You have one (1) remaining review left. As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), when you reach to the limit of 3 code reviews and your project was not approved, you will need to repeat the project. If you think that the code review was not fair, you can [request a second-opinion using this form.](https://airtable.com/shrmBIv6cXYrq4Fgv)**

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Comment by chukwuemeka1234** [comment]
I need help pls. Bereket Beshane

**Comment by Berabjesus** [comment]
> I need help pls. Bereket Beshane

How may i help you @chukwuemeka1234

**Review by george-shammar — APPROVED** [review]
### **STATUS: APPROVED** :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

**This milestone is approved** **Thank you for implementing the code review feedbacks from the previous reviewers. Good job**. :+1: :+1: :clap: :clap: 

**Congratulations**. :wine_glass: :wine_glass: 

**I like your website, it looks neat**. :clap: :clap: 


[OPTIONAL]

- [ ] **I strongly recommend that you clean up the .gtihub directory. You currently have asset directory nested in it with numerous image files.** 


**You can go ahead and merge this branch to the main branch.**

**Cheers and good luck with your next milestone.** :wine_glass: :wine_glass:

**Pull request merged**